README.md

```diff
@@ -1,13 +1,16 @@
 # Project Happy Thoughts API
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This week's project was to use Express and MongoDB to build an API which includes both GET request endpoints to return data and POST endpoints to create data. It is now also connected to the week 7 project "Happy Thoughts".
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+We got exact instructions on endpoints:
```

Comment:
Nice to include a detail of the endpoint requirements in your ReadMe, helps to explain the choices made in the code.